teams/README.md

```diff
@@ -0,0 +1,12 @@
+# Teams and projects
+
+The Tilt team is split into multiple (currently 2 teams) that each work on an individual project, typically for a 5 or 6 weeks sprint, in a synced cadence. At the end of the sprint, teams are disbanded and new teams and projects are formed for the following sprint.
+
+A project has a single charter outlining the business background and motivation for the upcoming sprint. It also contains a set of preliminary goals, which serve as success criteria for the project. Once a team is assigned to a project, they are responsible for further definining the goals, aiming to make them more objective and measurable. They are also responsible for scoping out work for the duration of the sprint. Teams are expected to move quickly and be responsive to changing user needs. As such, the charter (including it's goals and work scope) may change during the course of the sprint.
+
+A team is empowered to make any changes to fulfill the goals. So even though the work scope typically contains product changes, it can also encompass other changes such as updating the marketing site, setting up new user-facing processes, or working with external partners.
+
+Project work and team collaboration should be organized thusly:
+- There should be a Google doc for the project charter. For example, see [this doc](https://docs.google.com/document/d/1Sjd67pAHABTII00OKPbbK62K20J35NLGiZck3l7itpg/edit).
+- Candidate [Clubhouse stories](http://app.clubhouse.io/windmill/) should be labeled with a label named `proj-team-name`.
+- There should be a Slack channel called `proj-team-name`.
```

Comment:
I like this! Maybe a bullet for how we keep the team as a whole apprised of progress?

> Monday mornings, teams meet and plan their work for the week. During all-hands planning, each team gives a brief summary of their plan for the week.

(or something)

Reply:
I'm not a huge fan of being too prescriptive with process and prefer being descriptive of desired outcomes. But this may be unavoidable. I'm not going to add anything here in this PR, (so that we can merge quickly), but I expect we can discuss adding more details of how we do standups/planning in separate PRs.